controlplane/pkg/model/base.go

```diff
@@ -39,30 +40,38 @@ func (b *baseDomain) load(key string) (interface{}, bool) {
 	return v.clone(), true
 }
 
-func (b *baseDomain) store(key string, value cloneable) {
+func (b *baseDomain) store(key string, value cloneable, update bool) bool {
 	b.mtx.Lock()
 	defer b.mtx.Unlock()
 
 	old, exist := b.innerMap[key]
 	if !exist {
```

Comment:
What if `update = true`, should we return `false` at that point?

Reply:
No, `update = true` means `AddOrUpdate`.